dimod/quadratic/quadratic_model.py

```diff
@@ -444,34 +467,33 @@ def add_quadratic_from(self, quadratic: Union[Mapping[Tuple[Variable, Variable],
 
     @forwarding_method
     def add_variable(self, vartype: VartypeLike,
-                     v: Optional[Variable] = None,
+                     label: Optional[Variable] = None,
                      *, lower_bound: int = 0, upper_bound: Optional[int] = None) -> Variable:
         """Add a variable to the quadratic model.
 
         Args:
             vartype:
                 Variable type. One of:
 
-                * :class:`.Vartype.SPIN`, ``'SPIN'``, ``{-1, 1}``
-                * :class:`.Vartype.BINARY`, ``'BINARY'``, ``{0, 1}``
-                * :class:`.Vartype.INTEGER`, ``'INTEGER'``
+                * :class:`~dimod.Vartype.SPIN`, ``'SPIN'``, ``{-1, 1}``
+                * :class:`~dimod.Vartype.BINARY`, ``'BINARY'``, ``{0, 1}``
+                * :class:`~dimod.Vartype.INTEGER`, ``'INTEGER'``
 
             label:
-                A label for the variable. Defaults to the length of the
+                Label for the variable. Defaults to the length of the
                 quadratic model, if that label is available. Otherwise defaults
                 to the lowest available positive integer label.
 
             lower_bound:
-                A lower bound on the variable. Ignored when the variable is
-                not :class:`Vartype.INTEGER`.
+                Lower bound on the variable. Ignored when the variable is
```

Comment:
Let's save ourselves some effort later and say `Ignored when the variable is BINARY or SPIN`.